### ISSUE TITLE
Error when importing @clockwork-xyz/sdk module

### ISSUE THREAD

**Issue by mwrites** [clockwork-xyz/sdk#1]
How to import for the clockwork typescript module in a fresh project? I have tried importing it in a fresh anchor project (from the typescript tests folder).

### Config

```bash
solana --version: solana-cli 1.14.12 (src:979792ba; feat:4145199441)
anchor --version: anchor-cli 0.26.0
node --version: v16.14.0
```

### How to reproduce?

Reference project: [https://github.com/mwrites/test-ts-sdk](https://github.com/mwrites/test-ts-sdk)

1. `anchor create test-ts-sdk && cd test-ts-sdk`
2. `yarn add @clockwork-xyz/sdk@0.2.0`
3. Try to import the clockwork module in `tests/test-ts-sdk.ts`

```ts
import * as anchor from "@project-serum/anchor";
import { Program } from "@project-serum/anchor";
import { TestTsSdk } from "../target/types/test_ts_sdk";
import { ClockworkProvider } from "@clockwork-xyz/sdk";

describe("test-ts-sdk", () => {
  // Configure the client to use the local cluster.
  const provider = anchor.AnchorProvider.env();
  anchor.setProvider(provider);
  const program = anchor.workspace.TestTsSdk as Program<TestTsSdk>;
  const clockworkProvider = new ClockworkProvider(provider.wallet, provider.connection);

  it("Is initialized!", async () => {
    const [thread] = clockworkProvider.getThreadPDA(provider.wallet.publicKey, "foo");
  });
});

```
4. `anchor test` throws the below error 👇

```bash
$ /Users/mwrites/Downloads/test-ts-sdk/node_modules/.bin/ts-mocha -p ./tsconfig.json -t 1000000 'tests/**/*.ts'

Error: Cannot find module '/Users/mwrites/Downloads/test-ts-sdk/node_modules/@clockwork-xyz/sdk/lib/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (node:internal/modules/cjs/loader:353:19)
    at Function.Module._findPath (node:internal/modules/cjs/loader:566:18)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/mwrites/Downloads/test-ts-sdk/tests/test-ts-sdk.ts:4:1)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Module.m._compile (/Users/mwrites/Downloads/test-ts-sdk/node_modules/ts-node/src/index.ts:439:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/mwrites/Downloads/test-ts-sdk/node_modules/ts-node/src/index.ts:442:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.exports.requireOrImport (/Users/mwrites/Downloads/test-ts-sdk/node_modules/mocha/lib/nodejs/esm-utils.js:60:20)
    at Object.exports.loadFilesAsync (/Users/mwrites/Downloads/test-ts-sdk/node_modules/mocha/lib/nodejs/esm-utils.js:103:20)
    at singleRun (/Users/mwrites/Downloads/test-ts-sdk/node_modules/mocha/lib/cli/run-helpers.js:125:3)
    at Object.exports.handler (/Users/mwrites/Downloads/test-ts-sdk/node_modules/mocha/lib/cli/run.js:374:5)
```

**Comment by vicyyn** [comment]
Looking into this. Might be a problem with the package.json or tsconfig.json

**Comment by vicyyn** [comment]
Fixed as of v0.2.3
